__tests__/index.spec.js

```diff
@@ -139,22 +156,39 @@ describe('js-to-styles-vars-loader', () => {
             context: path.resolve(),
             addDependency () {}
         };
+        const content = "require('./mocks/colors.js');\n" +
+            ".someClass { color: #fff;}";
+
+        const trimmer = (str) => {
+          return (str.split("\n").filter(a => a).map(s => s.trim()).join(" ")).trim()
```

Comment:
That filter with identity function seems unnecessary.

Reply:
Yes I think it is, has been left in by accident.